### PR TITLE
Pin LXD dependency to `stable-5.21`

### PR DESCRIPTION
**Pull request by roosterfish**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -45,7 +45,11 @@ endif
 # Update targets.
 .PHONY: update-gomod
 update-gomod:
-	go get -t -v -d -u ./...
+	go get -t -v -u ./...
+
+	# Static pins
+	go get github.com/canonical/lxd@stable-5.21 # Stay on v2 dqlite
```
Comment (tomponline) [inline Makefile]: "and LXD LTS client" ?

**Review by tomponline — DISMISSED** [review]

**Comment by roosterfish** [comment]
Mhhh looks we have to revert the changes to the `Response`'s `Render(w http.ResponseWriter) error` as `@stable-5.21` doesn't contain the changes from https://github.com/canonical/lxd/pull/13825 which we already had in `microcluster/v2`.

But as it only affects the internal package it shouldn't be too much of an issue either.

File: Makefile (hunk 1 above)
Comment (roosterfish) [reply Makefile]: Fixed

**Comment by tomponline** [comment]
> Mhhh looks we have to revert the changes to the `Response`'s `Render(w http.ResponseWriter) error` as `@stable-5.21` doesn't contain the changes from [canonical/lxd#13825](https://github.com/canonical/lxd/pull/13825) which we already had in `microcluster/v2`.
> 
> But as it only affects the internal package it shouldn't be too much of an issue either.

I can backport if needed, which commit do you need?

**Comment by tomponline** [comment]
If its only internal, feel free to revert for now, but be warned it will make it into stable-5.21 eventually.

**Review by tomponline — APPROVED** [review]

**Comment by roosterfish** [comment]
@tomponline I have quickly resolved the required chain of commits:

* https://github.com/canonical/lxd/pull/13825/commits/ba31a90cfdb86ecabacc5f23e1c34e5bedbbf297
* https://github.com/canonical/lxd/pull/13825/commits/9143502719344473f7b912060f04f4b51c79d123
* https://github.com/canonical/lxd/pull/13825/commits/815cdb2a8c9b72ca19d63bb61b7f64cb64fce459

The first produces a conflict because https://github.com/canonical/lxd/pull/13825/commits/7798c53f6d825c020cd11152e442e80dc0199803 potentially already got backported.

**Pull request merged**